### ISSUE TITLE
resizable-arraybuffer and growable-sharedarraybuffer should be removed in favor of resizable-buffers

### ISSUE THREAD

**Issue by syg** [web-platform-dx/web-features#2254]
I raised an issue some time ago that the `resizable-arraybuffer` and `growable-sharedarraybuffer` features should be combined into a single feature. That has resulted in `resizable-buffers`, but now we just have all 3. :)

Let's remove `resizable-arraybuffer` and `growable-sharedarraybuffer`.

**Comment by atopal** [comment]
cc @Elchi3

**Comment by Elchi3** [comment]
I removed them in https://github.com/web-platform-dx/web-features/pull/1605.

Where do they still show up?

**Comment by syg** [comment]
> I removed them in https://github.com/web-platform-dx/web-features/pull/1605.

> Where do they still show up?

On webstatus.dev, both on the main page, https://webstatus.dev/features/resizable-arraybuffer, and https://webstatus.dev/features/growable-sharedarraybuffer

**Comment by Elchi3** [comment]
@atopal I guess this is more an issue for https://github.com/GoogleChrome/webstatus.dev

**Comment by atopal** [comment]
@jcscottiii I'm assuming this is an issue, because we're not handling merged features yet?

**Comment by jcscottiii** [comment]
> @jcscottiii I'm assuming this is an issue, because we're not handling merged features yet?

@atopal Essentially, yes. Also, on webstatus.dev, we do not automatically delete feature IDs if they disappear in the web-features repository. We can do that if that's desired though. 

I have marked those two feature IDs to be excluded from now on. 

Also, @ddbeck. I think #1605 should have been a breaking change and moved this from v1.x.x to v2.x.x according to this, right?

https://github.com/web-platform-dx/web-features/blob/ec1ff12b3afcdc43331a38d2c9be05cdb2b747ff/docs/publishing.md?plain=1#L14

**Comment by ddbeck** [comment]
> Also, @ddbeck. I think #1605 should have been a breaking change and moved this from v1.x.x to v2.x.x according to this, right?

@jcscottiii Yes, however we didn't document or start doing semver until after that PR, with [v1.3.0](https://github.com/web-platform-dx/web-features/releases/tag/v1.3.0).

Since 1.0.0, we've had these removals:

- canvas-2d-color-managment
- canvas-text
- canvas-text-baselines
- gravity-sensor
- growable-sharedarraybuffer
- resizable-arraybuffer

If those still exist on web.dev, I'd suggest pruning them. And if we remove features at later major releases, I'd suggest dropping those too. But hopefully that will be rare (or we'll have some data structure to indicate renames, merges, etc).

**Comment by jcscottiii** [comment]
Thanks for that list @ddbeck!
